builds/jazz-build-module/aws-apigateway-module.groovy

```diff
@@ -75,6 +75,17 @@ def injectLambdaIntegration(method, filePath) {
   }
 }
 
+def getApiGatewayCore(accountDetailsPrimary){
+  def gatewayValueInfo
+  for (item in accountDetailsPrimary.REGIONS) {
+    if(item.PRIMARY){
+      gatewayValueInfo = item.API_GATEWAY
+      break
+    }
+  }
+  return getAPIIdNameMapping(gatewayValueInfo.PROD, 'jazz', '*')
```

Comment:
Should this always be PROD?

Reply:
Yes, actually this is for constructing the base_url where prod_api_id is required